frontend/src/components/regions/CardDetails/utils.js

```diff
@@ -0,0 +1,3 @@
+export const trimLongReview = (review) => {
+  return review.split("\n").join("");

```

Comment:
If my review looks like this:

```
A whole
lot of
stuff blah blah blah
```

Then trimming it should not result in:

```
A wholelot ofstuff blah blah blah
```

So instead of trimming it, we are just concatinating all the lines and not leaving any spaces between some words. I'm not sure that this is an improvement. 

The trimmed version should look more like:

```
A whole lot of st...
```

Notice that this is actually shorter than the original text. Also the words aren't glued together, there is still whitespace.